RecoParticleFlow/PFClusterProducer/python/particleFlowClusterHBHETimeSelected_cfi.py

```diff
@@ -9,112 +9,112 @@
             minEnergy = cms.double(0.0),
             maxEnergy = cms.double(2.0),
             endcap = cms.bool(False),
-            minTime = cms.double(-22.),
-            maxTime = cms.double(22.)
+            minTime = cms.double(-7.6),
```

Comment:
thanks, you are right, i will make the change right away. Do i make another pull request, or I can make the change here directly?